index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Expobavio</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+  <nav class="navbar navbar-expand-lg navbar-light bg-white fixed-top">
```

Comment:
- [ ] The top dark menu section missing on both pages. Please implement this section and be sure : 

- The social media links, logout, and language links are aligned to the right.
- Section should disappear in mobile.

| Design Template | Project |
|-------------------|--------|
| ![Screenshot from 2021-09-09 22-19-47](https://user-images.githubusercontent.com/1642014/132749370-3ceac04f-19ea-49fa-921f-05929662ed93.png) | ![Screenshot from 2021-09-09 22-09-11](https://user-images.githubusercontent.com/1642014/132749398-eb26768e-99cc-46bb-a9e8-a1aaaca7aca7.png) |